src/data/term-result-mappings.ts

```diff
@@ -87,6 +87,97 @@ const resultMapping: ResultMapping = {
   ],
   condition: ['reference/operator/aggregation/cond', 'tutorial/query-documents', 'reference/operator/query/expr'],
   relation: ['applications/data-models', 'introduction', 'core/data-modeling-introduction'],
+  cond: ['reference/operator/aggregation/cond', 'tutorial/query-documents', 'reference/operator/query/expr'],
+  node: ['quick-start/download-and-install', 'fundamentals/promises'],
+  'node.js driver': ['quick-start/download-and-install', 'fundamentals/promises'],
+  nodejs: ['quick-start/download-and-install', 'fundamentals/promises'],
+  $filter: [
+    'reference/operator/aggregation/filter',
+    'reference/operator/aggregation',
+    'reference/operator/projection/positional',
+  ],
+  $match: ['reference/operator/aggregation/match', 'core/aggregation/', 'core/aggregation-pipeline-optimization/'],
```

Comment:
Just discovered that core/aggregation isn't a page anymore - it's at /aggregation